go.mod

```diff
@@ -42,9 +32,9 @@ require (
 	k8s.io/client-go v0.21.4
 	k8s.io/code-generator v0.21.4
 	k8s.io/kube-openapi v0.0.0-20210305001622-591a79e4bda7
-	k8s.io/utils v0.0.0-20210111153108-fddb29f9d009 // indirect
 	knative.dev/caching v0.0.0-20211123074749-2c27e22a9971
 	knative.dev/hack v0.0.0-20211122162614-813559cefdda
+	knative.dev/kperf v0.0.0-20211123152158-e478958fd2da
```

Comment:
This'll cause circular dependencies - https://github.com/knative-sandbox/kperf/blob/main/go.mod#L19, but I don't have a good solution for it..

Reply:
looking to see if there is refactoring in kperf, otherwise will just use the binary in the test

Reply:
switched to using the kperf binary